lib/manager/bundler/extract.ts

```diff
@@ -184,11 +185,19 @@ export async function extractPackageFile(
   if (!res.deps.length && !res.registryUrls.length) {
     return null;
   }
+
   if (fileName) {
     const gemfileLock = fileName + '.lock';
     const lockContent = await platform.getFile(gemfileLock);
     if (lockContent) {
       logger.debug({ packageFile: fileName }, 'Found Gemfile.lock file');
+      const lockedEntries = extractLockFileEntries(lockContent);
+      res.deps.forEach((dep, index) => {
```

Comment:
Is this `index` approach in order to avoid eslint warnings for `for dep of deps` syntax?

Reply:
No. My intentions were to modify each of those objects directly.

Reply:
Any reason not to skip using `index` and instead just do `dep.lockedVersion = lockedDepValue` ?

Reply:
Not any specific reason. I prefer to make changes it array through the index. I guess I can remove the index thing.